api/tests/quest/integration/infrastructure/repositories/combined-course-participation-repository_test.js

```diff
@@ -145,21 +147,34 @@ describe('Quest | Integration | Infrastructure | repositories | Combined-Course-
   });
 
   describe('#findByCombinedCourseIds', function () {
-    it('should return participations for given quest IDs', async function () {
+    it('should return a paginated list ofparticipations for given quest IDs', async function () {
```

Comment:
```suggestion
    it('should return a paginated list of participations for given quest IDs', async function () {
```
*todo*: typo à corriger